### ISSUE TITLE
Clear out more default settings from AssemblyInfo

### ISSUE THREAD

**Issue by mungojam** [hvanbakel/CsprojToVs2017#166]
There seem to be a large number of different assembly options and different projects end up with different default entries in there. It would be good to pick these up so that the AssemblyInfo gets deleted more often. I expect they don't normally need to be put into the project file if they are just default

Probably fine to remove:
```csharp
[assembly: AssemblyConfiguration("")]
[assembly: AssemblyTrademark("")]
[assembly: AssemblyCulture("")]
```

Not sure:
```csharp
[assembly: NeutralResourcesLanguage("en")]
// Setting ComVisible to false makes the types in this assembly not visible 
// to COM components.  If you need to access a type in this assembly from 
// COM, set the ComVisible attribute to true on that type.
[assembly: ComVisible(false)]

// The following GUID is for the ID of the typelib if this project is exposed to COM
[assembly: Guid("...some-unique-guid...")]
```

**Comment by mungojam** [comment]
I'm now handling these in https://github.com/hvanbakel/CsprojToVs2017/pull/167

```
[assembly: AssemblyConfiguration("")]
[assembly: AssemblyTrademark("")]
[assembly: AssemblyCulture("")]
```

Meanwhile `[assembly: NeutralResourcesLanguage("en")]` is now just moved to csproj file. I don't think "en" is the default anyway.

`ComVisible` is a bit of an unknown as I think the default is actually `true` if the attribute isn't there, but VS adds it as `false` by default.

I'll raise as a separate issue

**Comment by mungojam** [comment]
The only remaining one I know about is ComVisible which I've raised as a separate issue so I think this one can be closed as fixed in #167
